src/components/Calculator.js

```diff
@@ -1,8 +1,10 @@
 import '../App.css';
 import CalculatorKeys from './CalculatorKeys';
+import DisplayQuote from './DisplayQuotes';
```

Comment:
- [ ] The quotes components should be imported into the app component, not the calculator. Kindly move this import.